examples/flutter_gallery/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java

```diff
@@ -1,13 +1,13 @@
 package io.flutter.plugins;
 
 import io.flutter.plugin.common.PluginRegistry;
-import io.flutter.plugins.urllauncher.UrlLauncherPlugin;
+import io.flutter.plugins.url_launcher.UrlLauncherPlugin;
```

Comment:
Is your local repository at head? This has been renamed to not use the underscore a while ago: https://github.com/flutter/plugins/blob/master/packages/url_launcher/pubspec.yaml#L10

Reply:
Yes, I'm at head.